tests/hlsl-intrinsic/helper-lane.slang

```diff
@@ -0,0 +1,13 @@
+//TEST:SIMPLE(filecheck=HLSL): -entry main -target hlsl -stage fragment
+//TEST:SIMPLE(filecheck=GLSL): -entry main -target glsl -stage fragment
+//TEST:SIMPLE(filecheck=METAL): -entry main -target metal -stage fragment
+//TEST:SIMPLE(filecheck=SPIRV): -entry main -target spirv -stage fragment
```

Comment:
Let's change the target from spirv to spirv-asm.
Or you can simply add one more,
```
//TEST:SIMPLE(filecheck=SPIRV): -entry main -target spirv -stage fragment
//TEST:SIMPLE(filecheck=SPIRV): -entry main -target spirv-asm -stage fragment
```